Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
```

Comment:
variables should be named in camelCase https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names